examples/effects_clipping_caps.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
```

Comment:
Sounds reasonable. So probably three examples: 
1. Axis-aligned clipping planes with a cube being the capNode. No artifact
2. Arbitrary clipping plane, no intersection. No artifact
3. Arbitrary clipping plane with intersection. Artifacts and info text. 
I might return to this later cuz I can still work on this after my internship